cmd/thriftrw-list-deps/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"go.uber.org/thriftrw/compile"
+)
+
+type args struct {
+	ThriftFile string `positional-arg-name:"thrift-file"`
+}
+
+var opts struct {
+	Relative string `long:"relative" description:"Output paths will be relative to this directory"`
```

Comment:
nit:

    If specified, output paths will be relative to this directory.

Reply:
Also, can we call this `--relative-to`? So that usage looks like,

    thriftrw-list-deps --relative-to=idl idl/whatever/foo.thirft